cmd/juju/application/deploy_test.go

```diff
@@ -1446,24 +1423,19 @@ func (s *DeploySuite) TestDeployLocalWithSeriesMismatchReturnsError(c *gc.C) {
 
 	_, _, err := s.runDeployWithOutput(c, charmDir.Path, "--base", "ubuntu@12.10")
 
-	c.Check(err, gc.ErrorMatches, `terms1 is not available on the following series: quantal not supported`)
+	c.Check(err, gc.ErrorMatches, `terms1 is not available on the following base: ubuntu@12.10/stable`)
```

Comment:
`not supported` is no longer in the error message because `CharmBase` returns a `errors.NewNotFound(nil, ...)` rather than `errors.NotFoundf(...)` for some reason